arangod/Graph/KShortestPathsFinder.h

```diff
@@ -165,6 +165,7 @@ class KShortestPathsFinder : public ShortestPathFinder {
       _frontier.insert(centre, std::make_unique<DijkstraInfo>(centre));
     }
     ~Ball() = default;
+    const VertexRef centre() const { return _centre; };
```

Comment:
We generally use American English in the code base. Consider renaming it to `center`.

Reply:
Yes we do, but well it does not bother me too much.
I do not mind too much if this is not addressed here.
Technically this is working